packages/controller/lib/setup.js

```diff
@@ -1121,17 +1121,21 @@ async function processCommand(command, args, params, callback) {
                     `Command "clean" clears all Objects and States. To execute it write "${tools.appName} clean yes"`
                 );
             } else {
-                dbConnect(params, (_obj, _stat, isNotRun) => {
+                dbConnect(params, async (_obj, _stat, isNotRun) => {
                     if (!isNotRun) {
                         console.error(`Stop ${tools.appName} first!`);
                         return void callback(EXIT_CODES.CONTROLLER_RUNNING);
                     }
-                    cleanDatabase(true, count => {
+
+                    try {
+                        const count = await cleanDatabase(true);
                         console.log(`Deleted ${count} states`);
-                        restartController();
-                        console.log(`Restarting ${tools.appName}...`);
-                        return void callback();
-                    });
+                    } catch {
+                        // ignore
+                    }
+                    restartController();
+                    console.log(`Restarting ${tools.appName}...`);
+                    return void callback();
```

Comment:
```suggestion
                    callback();
```
return void at the end of a function doesn't make a ton of sense.

Reply:
not touched but yes, can simply just call cb